docs/import-data/data-streams/manage-streams-lab.md

```diff
@@ -0,0 +1,178 @@
+---
+id: manage-streams-lab
+title: Connect to data streams using Memgraph Lab wizard
+sidebar_label: Connect using Memgraph Lab wizard

```

Comment:
I would drop wizard in all places, only if it is not officali called like that.

Reply:
1. wizard... we need something then because using Memgraph Lab is not clarifying that you are not using queries because you can also connect to data streams using memgraph lab AND queries
2. what do you mean it OVERLAPS? it should just break into another row (I have 2 rows in total)